community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This CSV is missing a `spec.install` section that defines the deployment of the operator and any Roles and ClusterRoles that it's service account may require.

Reply:
Could you share an example or template we can reference for this section? Our Operator is a helm chart based one, not constructed using the Operator SDK. Thanks.

Reply:
Sure thing! You can use the etcd operator CSV as a template: https://github.com/operator-framework/community-operators/blob/master/community-operators/etcd/etcdoperator.v0.9.2.clusterserviceversion.yaml

Reply:
Jaeger is also a good template: https://github.com/operator-framework/community-operators/blob/master/community-operators/jaeger/jaeger.v1.8.2.clusterserviceversion.yaml